examples/platform/silabs/SiWx917/SiWx917/wfx_rsi.h

```diff
@@ -28,11 +28,9 @@
 #define WFX_RSI_TASK_SZ (1024 + 1024)           /* Stack for the WFX/RSI task		*/
 #define WFX_RSI_BUF_SZ (1024 * 10)              /* May need tweak 			*/
 #define WFX_RSI_CONFIG_MAX_JOIN (5)             /* Max join retries			*/
+// TODO: Default values are usually in minutes, but this is in ms. Confirm if this is correct
+#define WFX_RSI_DHCP_POLL_INTERVAL (250) /* Poll interval in ms for DHCP		*/
```

Comment:
is this 250 ms poll interval when trying to acquire the IP address? or is this an extended period of time?

Reply:
From the looks of the code, this is actually the frequency at which we handle the polling, checking if we were able to renew our lease. We will need to experiment modifying this frequency.

We do it upon connection or disconnection.